src/ServicePulse.Host/vue/src/views/DashboardView.vue

```diff
@@ -27,13 +29,13 @@ import { licenseStatus } from "@/composables/serviceLicense";
               <div class="col-12">
                 <div class="row">
                   <div class="col-4">
-                    <DashboardItem :counter="stats.number_of_failed_heartbeats" :url="'/a/#/endpoints'" :iconClass="'fa-heartbeat'">Heartbeats</DashboardItem>
+                    <DashboardItem :counter="stats.number_of_failed_heartbeats" :url="routeLinks.heartbeats" :iconClass="'fa-heartbeat'">Heartbeats</DashboardItem>
                   </div>
                   <div class="col-4">
-                    <DashboardItem :counter="stats.number_of_failed_messages" :url="'#/failed-messages'" :iconClass="'fa-envelope'">Failed Messages</DashboardItem>
+                    <DashboardItem :counter="stats.number_of_failed_messages" :url="useLink({ to: routeLinks.failedMessage.root }).href.value" :iconClass="'fa-envelope'">Failed Messages</DashboardItem>
```

Comment:
This one is a bit more code than what we would expect!
This is because at the moment `DashboardItem` uses use a `a href` link instead of `RouterLink`.
Once we migrate all pages ut of Angular, we can change `DashboardItem` to use `RouterLink` and hence no need to use the `useLink` to give us a fully qualified url.